src/front-end/src/components/MenteeFormAndEmbedder/MenteeFormAndEmbedder.js

```diff
@@ -12,24 +13,41 @@ function MentorFormAndEmbedder() {
     <div>
       <div>
         <MenteeForm />
-        <button onClick={embed} type="button">
-          Embed this form in your website
-        </button>
+        <center>
+          <h1>
+            Click on the &quot;Generate iFrame Tag&quot; button to create an
+            iFrame tag that can be embedded into your Website!
+          </h1>
+        </center>
+        <center>
+          <Button
+            onClick={embed}
+            type="primary"
+            htmlType="submit"
+            className="formSubmitButton"
+            size="large"
+          >
```

Comment:
I think adding a different colour to the button would add some variance to the site. In my design doc, I used green, but since the background colour of the iFrame textarea is a shade of purple, maybe the button could be that same colour (or both could be an entirely different colour, but it might be nice if both the button and the textarea match). We should probably also change the class name of the button so it's more accurate
```suggestion
          <Button
            onClick={embed}
            type="primary"
            htmlType="submit"
            className="generateiFrameButton"
            size="large"
            style={{ backgroundColor: '#110a47', border: 'none' }}
          >
```

Reply:
Changed color